pkg/sql/sem/tree/datum.go

```diff
@@ -3891,6 +4061,12 @@ func NewDefaultDatum(evalCtx *EvalContext, t *types.T) (d Datum, err error) {
 		return dNullJSON, nil
 	case types.TimeTZFamily:
 		return dZeroTimeTZ, nil
+	case types.GeometryFamily, types.GeographyFamily:
```

Comment:
I think you have to put a value for this - it doesn't matter what. This was added recently to work around an issue with column backfills, I think...